config/locales/en.yml

```diff
@@ -45,8 +45,8 @@ en:
           If many people have already RSVP'd, consider sending a message out to this bus list asking for interest as well.
         notes: Notes are shared with applicants. Supports Markdown and HTML.
       user:
-        role: Limited access prevents the admin from adding, modifying, or deleting any records; modifications through the check-in process are allowed. Event tracking limits to only event tracking.
-        is_active: Deactivating a user will prevent them from logging in. Their access will be immediately revoked from the admin and application pages.
+        role: Limited access prevents the director from adding, modifying, or deleting any records; modifications through the check-in process are allowed. Event tracking limits to only event tracking.
```

Comment:
Sorry this was a little confusing, role should read: "Organizers are prevented from adding, modifying, or deleting any records; modifications through the check-in process are allowed. Volunteers only have access to the check-in process."